sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -202,12 +202,12 @@ int _gnrc_rpl_dodag_show(void)
 
         cleanup = dodag->instance->cleanup < 0 ? 0 : dodag->instance->cleanup;
 
-        printf("\tdodag [%s | R: %d | OP: %s | PIO: %s | CL: %" PRIi8 "s | "
+        printf("\tdodag [%s | R: %d | OP: %s | PIO: %s | CL: %ds | "
                "TR(I=[%d,%d], k=%d, c=%d, TC=%" PRIu32 "s, TI=%" PRIu32 "s)]\n",
                ipv6_addr_to_str(addr_str, &dodag->dodag_id, sizeof(addr_str)),
                dodag->my_rank, (dodag->node_status == GNRC_RPL_LEAF_NODE ? "Leaf" : "Router"),
                ((dodag->req_opts & GNRC_RPL_REQ_OPT_PREFIX_INFO) ? "on" : "off"),
-               cleanup, (1 << dodag->dio_min), dodag->dio_interval_doubl, dodag->trickle.k,
+               (int) cleanup, (1 << dodag->dio_min), dodag->dio_interval_doubl, dodag->trickle.k,
```

Comment:
Why is cleanup `int8_t` anyway? Due to alignment and the position it is allocated at on the stack it will probably waste 3 more bytes anyway.